src/hooks/useNotifications.ts

```
@@ -0,0 +1,224 @@
+
+      // Process events into notifications
+      const notifications: Notification[] = events.map(event => {
+        let type: NotificationType;
+        let eventId: string | undefined;
+        let groupId: string | undefined;
+        let content: string | undefined;
+
+        switch (event.kind) {
+          case 34550:
+            type = 'group_update';
+            const dTag = event.tags.find(tag => tag[0] === 'd');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)